CromIAM/src/test/scala/cromiam/webservice/WomtoolRouteSupportSpec.scala

```diff
@@ -11,11 +12,16 @@ import org.scalatest.matchers.should.Matchers
 class WomtoolRouteSupportSpec extends AnyFlatSpec with CromwellTimeoutSpec with Matchers with WomtoolRouteSupport with ScalatestRouteTest {
 
   override lazy val cromwellClient = new MockCromwellClient()
+  override lazy val samClient = new MockSamClient()
 
   behavior of "Womtool endpoint routes"
 
   it should "return 200 when we request to the right path" in {
-    Post(s"/api/womtool/v1/describe") ~> womtoolRoutes ~> check {
+    Post(
+      s"/api/womtool/v1/describe"
+    ).withHeaders(
+      List(Authorization(OAuth2BearerToken("my-token")), RawHeader("OIDC_CLAIM_user_id", "Zardoz@example.com"))
```

Comment:
Does the mock client check headers? If so could we have a "doesn't let through invalid users" test case?